SDAPackageWithVectorField/SDAWithVectorField/obstacle_map.py

```diff
@@ -0,0 +1,64 @@
+import numpy
+from math import atan2, cos, sin, pi, sqrt
+from SDAWithVectorField import *
+
+class ObstacleMap(object):
+	def __init__(self, drone_point, fly_zones):
+		self.obstacles = numpy.array([])
+		self.drone = Drone(drone_point, numpy.array([]))
+		self.fly_zones = FlightBoundariesContainer(fly_zones)
+		self.repulsive_forces = numpy.array([0,0,0])
+		self.attractive_force = numpy.array([0,0,0])
+		self.detection_radius = 0
+
+	def add_obstacle(self, obstacle):
+		if self.obstacles.size != 0:
+			final_obstacles = numpy.hstack([self.obstacles, obstacle])
+			self.obstacles = final_obstacles
+		else:
+			self.obstacles = numpy.array([obstacle])
+
+	def add_force(self):
```

Comment:
This method looks good, but we should change the name to be more descriptive to its function. How does ```update_repulsive_forces()``` sound?